site/docs/hive.md

```diff
@@ -17,113 +17,286 @@
+
+!!! Note
+    Due to the limitation of Hive `PARTITIONED BY` syntax, currently you can only partition by columns, 
+    which is translated to Iceberg identity partition transform.
+    You cannot partition by other Iceberg partition transforms such as `days(timestamp)`.
+
+The following Hive types have direct Iceberg types mapping:
+
+- boolean
+- float
+- double
+- integer
+- long
+- decimal
+- string
+- binary
+- date
+- timestamp (maps to Iceberg timestamp without timezone)
+- timestamplocaltz (Hive 3 only, maps to Iceberg timestamp with timezone)
+- struct
+- map
+- list
+
+The following Hive types are not supported by Iceberg:
+
+- interval_year_month
+- interval_day_time
+- union
+
+The following Hive types do not have direct Iceberg types mapping, but we can perform auto-conversion:
+
+| Hive type  | Iceberg type |
+| ---------- | ------------ |
+| byte       | integer      |
+| short      | integer      |
+| char       | string       |
+| varchar    | string       |
+
+You can enable this feature through Hadoop configuration (default not enabled):
+
+| Config key                               | Default                     | Description                                         |
+| -----------------------------------------| --------------------------- | --------------------------------------------------- |
+| iceberg.mr.schema.auto.conversion        | false                       | if CREATE TABLE should perform type auto-conversion |
+
+### DROP TABLE
+
+Tables can be dropped using the `DROP TABLE` command:
 
-#### Query the Iceberg table via Hive
-You should now be able to issue Hive SQL `SELECT` queries using the above table and see the results returned from the underlying Iceberg table.
 ```sql
-SELECT * from database_a.table_c;
+DROP TABLE [IF EXISTS] table_name [PURGE];
 ```
 
-### Features
+You can configure global purge behavior through Hadoop configuration:
+
+| Config key                  | Default                    | Description                                                     |
+| ----------------------------| ---------------------------| --------------------------------------------------------------- |
+| external.table.purge        | true                       | if all data and metadata should be purged in a table by default |
+
+Each table's default purge behavior can be further configured through table properties:
 
-#### Predicate pushdown
-Pushdown of the Hive SQL `WHERE` clause has been implemented so that these filters are used at the Iceberg TableScan level as well as by the Parquet and ORC Readers.
+| Property                    | Default                    | Description                                                       |
+| ----------------------------| ---------------------------| ----------------------------------------------------------------- |
+| gc.enabled                  | true                       | if all data and metadata should be purged in the table by default |
```

Comment:
@marton-bod: Do we have the PR merged where we linked the gc.enabled and the external.table.purge properties?

Reply:
Yep, it's merged, so now they should change in tandem in these two scenarios:

- Changing `gc.enabled` on the Iceberg table via `UpdateProperties` -> `external.table.purge` updated on HMS table accordingly
- Setting `external.table.purge` as a table prop during Hive CREATE TABLE -> `gc.enabled` is pushed down accordingly to the Iceberg properties as well

What's missing is the Hive `ALTER TABLE SET TBLPROPERTIES` scenario. In this case, changing `external.table.purge` via this command does not update `gc.enabled` on the Iceberg table. This is a limitation on Hive 3.1.2 because the `HiveMetaHook` doesn't have all the hooks for alter tables (which we have recently merged to Hive master).

Reply:
Added some explanation based on what Marton suggested, please check if that looks good to you.